tests/containers/containers.test.js

```diff
@@ -127,15 +127,16 @@ describe('Service Lifecyle Integration Test', () => {
   });
 
   it('should deploy updated service/container to scaleway', () => {
-    replaceTextInFile('my-container/server.py', 'Hello, World from Scaleway Container !', 'Container successfully updated');
```

Comment:
I think you should add additional test for this an not change the current behaviour, wdyt ?